pkg/operator/apis/monitoring/v1alpha1/types.go

```diff
@@ -169,6 +169,9 @@ type GrafanaAgentSpec struct {
 	// config port 8080 on the agent.
 	// +kubebuilder:default=false
 	EnableConfigReadAPI bool `json:"enableConfigReadAPI,omitempty"`
+
+	// disableReporting disable reporting of enabled feature flags to Grafana.
+	DisableReporting bool `json:"disableReporting,omitempty"`
```

Comment:
Just curious, but what is the `+kubebuilder:default` directive that exists on some fields (eg. on EnableConfigReadAPI) above? Is it something autogenerated?

Reply:
Good that you pointed out! This set values when running `make crd`. I will also add the directive here to default to false